assets/src/components/mapPage.tsx

```diff
@@ -38,6 +38,10 @@ const thereIsAnActiveSearch = (
   searchPageState: SearchPageState
 ): boolean => vehicles !== null && searchPageState.isActive
 
+const HorizontalSeparator = () => (
+  <hr className="c-map-page__horizontal-separator" />
+)
```

Comment:
Oh, the reason I didn't was because that would change the semantics (`<hr>` -> `<img>`) and I didn't think we'd be ready for an in-depth conversation about that at this time

Reply:
@mbta/skate-developers as Kayla said, this is sortof a continutation of this [github PR comment thread](https://github.com/mbta/skate/pull/2086#discussion_r1228336815). Do y'all have any issues with using `VisualSeparator` and changing from an `<hr>` to an aria-ignored `<img>`?

Reply:
I'm rusty on the semantics, but `hr` did seem appropriate to me here based on your earlier question in the linked thread
> Does this count as a "section of content"?
My gut answer to that would be yes, but there may be some nuance I'm missing. I guess independent to the answer of that question, these two separators seem to be used in the same way so my preference is for consistency independent of which tag that means we should use. But I welcome other perspectives on that, especially since I still have a lot to learn about semantics!

Reply:
PTALA a4299c1462d543c928ee32cd5e359a2ef96d87b1 🙏🏻

Reply:
Okay sorry just catching up with the comment in #2086 because it came through after I merged the PR. I find it hard to really come down one way or another on the semantics of this element - it seems like ultimately it's at least partially a design question and overall just another consequence of not having a well-defined design system with standardized components. I'd say that I'm fine with either for now but ultimately we should try to push for more consistency across the application as opposed to trying to reason through it on a purely ad-hoc basis (though I know that's easier said than done!).